rpm/authselect.spec.in

```diff
@@ -236,8 +236,9 @@ if [ -f %{validfile} ]; then
     %__rm -f %{validfile}
 fi
 
-# Add nss-altfiles if we are on Kinoite or Silverblue
-if %__grep -Ei "kinoite|silverblue" /etc/os-release &> /dev/null; then
+# Add nss-altfiles if we are on an ostree/rpm-ostree based system.
+# Applies to Fedora IoT, Silverblue, Kinoite, CoreOS.
+if %__grep "OSTREE_VERSION=" /etc/os-release &> /dev/null; then
```

Comment:
Yeah, this is better.  

But we could also do `test -f /usr/lib/passwd` as a check for the presence of altfiles-style semantics, which I think would be even better because it's more clearly tied to altfiles - ostree is conceptually independent of altfiles (altfiles is one mechanism to deal with uid/gid drift, but not the only one).

Reply:
Would `/usr/lib/passwd` effectively exist at the time this %posttrans script would be run?

Reply:
Should yes, posttrans inherently runs after all files from content (except those dynamically generated by other posttrans) have run.

Reply:
From a side discussion, it might not exist if authselect is included in the base image thus we can not rely on this for the check.